project/LinkerdBuild.scala

```diff
@@ -368,15 +373,16 @@ object LinkerdBuild extends Base {
          |  GC_LOG="/var/log/namerd"
          |fi
          |
+         |# Check Java version for use in GC_LOG_OPTION and DEFAULT_JVM_OPTIONS
+         |LOCAL_JAVA_VERSION=$("${JAVA_HOME:-/usr}"/bin/java -version 2>&1 | sed 's/.*version "\([0-9]*\)\..*/\1/; 1q')
```

Comment:
why does "LOCAL" refer to?  why not just `JAVA_VERSION`?

Reply:
Answered here! https://github.com/linkerd/linkerd/pull/2272#discussion_r282295672